src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -2109,7 +2204,63 @@ private GencodeFuncotation createIgrFuncotation(final VariantContext variant,
+
+        // Set our version:
+        funcotationBuilder.setVersion(version);
+
+        // Set our data source name:
+        funcotationBuilder.setDataSourceName(getName());
+
+        return funcotationBuilder.build();
+    }
+
+    /**
+     * Creates a {@link GencodeFuncotation}s based on the given {@link Allele} with type
+     * {@link GencodeFuncotation.VariantClassification#FIVE_PRIME_FLANK} or
+     * {@link GencodeFuncotation.VariantClassification#THREE_PRIME_FLANK}
+     *
+     * @param variant The {@link VariantContext} associated with this annotation.
+     * @param altAllele The alternate {@link Allele} to use for this {@link GencodeFuncotation}.
+     * @param transcript {@link GencodeGtfTranscriptFeature} associated with this flank funcotation.
+     * @param gtfFeature {@link GencodeGtfGeneFeature} associated with this flank funcotation.
+     * @param reference The {@link ReferenceContext} in which the given {@link Allele}s appear.
+     * @param flankType {@link GencodeFuncotation.VariantClassification#FIVE_PRIME_FLANK} or
+     *                  {@link GencodeFuncotation.VariantClassification#THREE_PRIME_FLANK}
+     * @return A flank funcotation for the given allele
+     */
+    private GencodeFuncotation createFlankFuncotation(final VariantContext variant, final Allele altAllele, final GencodeGtfTranscriptFeature transcript, final GencodeGtfGeneFeature gtfFeature, final ReferenceContext reference, final GencodeFuncotation.VariantClassification flankType) {
+        final GencodeFuncotationBuilder funcotationBuilder = new GencodeFuncotationBuilder();
+
+        // NOTE: The reference context is ALWAYS from the + strand
+        final StrandCorrectedReferenceBases referenceBases = FuncotatorUtils.createReferenceSnippet(variant.getReference(), altAllele, reference, Strand.POSITIVE, referenceWindow);
+        final String referenceBasesString = referenceBases.getBaseString(Strand.POSITIVE);
+
+        final double gcContent = calculateGcContent(variant.getReference(), altAllele, reference, gcContentWindowSizeBases);
+
+        funcotationBuilder.setHugoSymbol(gtfFeature.getGeneName())
+                .setChromosome(variant.getContig())
+                .setStart(variant.getStart())
+                .setEnd(variant.getEnd())
+                .setStrand(transcript.getGenomicStrand())
+                .setVariantClassification(flankType)
+                .setVariantType(getVariantType(variant.getReference(), altAllele))
+                .setRefAllele(variant.getReference())
+                .setTumorSeqAllele2(altAllele.getBaseString())
+                .setGenomeChange(getGenomeChangeString(variant, altAllele))
+                .setAnnotationTranscript(transcript.getTranscriptId())
+                .setOtherTranscripts(gtfFeature.getTranscripts().stream().map(GencodeGtfTranscriptFeature::getTranscriptId).collect(Collectors.toList()))
```

Comment:
You shouldn't need to set other transcripts here - it'll get populated in `createGencodeFuncotationsByAllTranscripts` in the call to `populateOtherTranscriptsMapForFuncotation`.

Reply:
Fixed